jans-cedarling/cedarling/src/init/cedar_schema.rs

```diff
@@ -0,0 +1,86 @@
+/*
+ * This software is available under the Apache-2.0 license.
+ * See https://www.apache.org/licenses/LICENSE-2.0.txt for full text.
+ *
+ * Copyright (c) 2024, Gluu, Inc.
+ */
+
+use base64::prelude::*;
+
+#[derive(Debug, thiserror::Error)]
+pub enum ParceCedarSchemaErrMsg {
```

Comment:
typo, should be `ParseCedarSchemaErrMsg`.
also, please make it in full `ParseCedarSchemaError`, the `msg` prefix is not needed, that's quite the standard that's usually followed in Rust error handling.

Reply:
same like with `ParsePolicySetErrMsg`

Discussed that it is not error, but list of the messages.
Open to ideas how to make it more descriptive

Reply:
renamed to `ParceCedarSchemaSetMessage`